src/olympia/migrations/1149-db-schema-categories.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE `categories`
+    MODIFY `id` int(10) unsigned NOT NULL,
```

Comment:
`Cannot change column 'id': used in a foreign key constraint 'addons_categories_category_id_f4f5c093_fk_categories_id' of table 'olympia_dev_copy.addons_categories'`